examples/external-functions/serverless.yml

```diff
@@ -0,0 +1,32 @@
+service: mapping-templates
+provider:
+    name: aws
+    region: eu-west-1
+
+plugins:
+    - serverless-offline
+    - serverless-appsync-simulator
+    - serverless-appsync-plugin
+
+custom:
+    appSync:
+        name: Test
+        schema: schema.graphql
+        authenticationType: AWS_IAM
+        mappingTemplatesLocation: mapping-templates
+        mappingTemplates:
+            - dataSource: MyLambda
+              type: Query
+              field: getPosts
+              request: Query.getPosts.request.vtl
+              response: Query.getPosts.response.vtl
+        dataSources:
+            - type: AWS_LAMBDA
+              name: MyLambda
+              config:
+                  functionName: myLambda
+                  lambdaFunctionArn: "arn:aws:lambda:${self:provider.region}:*:getPosts-graphql"
+        functions:
```

Comment:
@EgorDm Please move this into `custom.appsync-simulator` since this is simulator related.

Reply:
I have moved the function config into the plugin (serverless-appsync-simulator) config.
It is indeed useful to add this separation.